Homework-13/script.js

```diff
@@ -0,0 +1,163 @@
+// task 1
+
+// Псевдослучайный генератор
+// Есть много областей, где нам нужны случайные данные.
+
+// Одной из них является тестирование. Нам могут понадобиться случайные данные: текст, числа и т.д., чтобы хорошо всё проверить.
+
+// В JavaScript мы можем использовать Math.random(). Но если что-то пойдёт не так, то нам нужно будет перезапустить тест, используя те же самые данные.
+
+// Для этого используются так называемые «сеяные псевдослучайные генераторы». Они получают «зерно», как первое значение, и затем генерируют следующее, используя формулу. Так что одно и то же зерно даёт одинаковую последовательность, и, следовательно, весь поток легко воспроизводим. Нам нужно только запомнить зерно, чтобы воспроизвести последовательность.
+
+// Пример такой формулы, которая генерирует более-менее равномерно распределённые значения:
+
+// next = previous * 16807 % 2147483647
+
+// Если мы используем 1 как зерно, то значения будут:
+// 1. 16807
+// 2. 282475249
+// 3. 1622650073
+// 4. …и так далее…
+
+// Задачей является создать функцию-генератор pseudoRandom(seed), которая получает seed и создаёт генератор с указанной формулой.
+// Пример использования:
+
+// function* pseudoRandom(seed) {
+//   let previous = 0;
+//   if (!previous) {
+//     previous = seed;
+//   }
```

Comment:
it's not obligatory
You call the function once, after that, you call "next" method of generator that will work only with "yield"

Reply:
You can simplify to `let previous = seed `